support/fixtures/factories/costCenter.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const factory = require('rosie').Factory;
+const faker = require('faker');
+
+factory
+  .define('costCenter')
+  .option('fromServer', false)
+  .attrs({
+    description: () => faker.hacker.phrase(),
+    id: () => faker.random.uuid(),
+    name: () => faker.commerce.productName(),
+    organizationId: () => factory.build('organization').id
```

Comment:
Should we also import `costCenterFacility` in here and have a `facilities` attribute with a `() => factory.buildList('costCenterFacility', 10)` or something like that?